src/common/is-node-attached.js

```diff
@@ -1,11 +1,11 @@
 // Marionette.isNodeAttached
 // -------------------------
 
-import Backbone from 'backbone';
+import domMixin from '../mixins/dom';
 
 // Determine if `el` is a child of the document
 const isNodeAttached = function(el) {
-  return Backbone.$.contains(document.documentElement, el);
+  return domMixin.containsEl(document.documentElement, el);
```

Comment:
Another reason to use el.parentNode is that Node.contains returns true when comparing a element to itself